ovos_tts_server/gradio_app.py

```diff
@@ -0,0 +1,67 @@
+import tempfile
+import gradio as gr
+
+from ovos_utils import LOG
+from neon_tts_plugin_coqui import CoquiTTS
+
+TTS = None
+
+
+def tts(text: str, language: str):
+    print(text, language)
+    # return output
+    with tempfile.NamedTemporaryFile(suffix=".wav", delete=False) as fp:
+        TTS.get_tts(text, fp, speaker={"language": language})
+        return fp.name
+
+
+def bind_gradio_service(app, tts_engine, title, description, info, badge,
+                        default_lang="en",):
+    global TTS
+    TTS = tts_engine
+
+    LOG.info(tts_engine.available_languages)
+    languages = list(tts_engine.available_languages or [default_lang])
+
+    # TODO: Below passed as args or from config
```

Comment:
this needs to come from some config file or from args, with generic default values if missing